src/app/shared/launch.service.ts

```diff
@@ -76,7 +76,23 @@ export abstract class LaunchService {
      * @param versionName The ToolVersion's name
      */
     getNextflowNativeLaunchString(workflowPath: string, versionName: string) {
-      return `$ nextflow run http://${workflowPath} -r ${versionName} -with-docker`;
+      return `$ nextflow run http://${workflowPath} -r ${versionName}`;
+    }
+
+    /**
+     * Gets local launch command
+     */
+    getNextflowLocalLaunchString() {
+      return `$ nextflow run nextflow.config`;
```

Comment:
Just curious, is there a separate ticket to run nextflow with dockstore like you can with cwl and wdl? That would allow the file provisioning plugins to be used like the are for the other languages. Or does it not make sense for Nextflow?

Reply:
A question well worth asking. We need to look at their engine to see if it makes sense or I think Andrew noted they have a parallel mechanism which would make this a bit redundant.

Reply:
Nextflow offers a lot of what the Dockstore CLI does, so it would be quite redundant. It also has more features that the Dockstore CLI doesn't have. For consistency though, a user should be able to use `dockstore workflow launch` for nextflow as well, even if it just downloads the dockstore workflow and launches it with nextflow without doing any fancy file provisioning.

Reply:
@denis-yuen Is this something you think is worth creating an issue for?